packages/extension/src/chromeServices/background/index.ts

```diff
@@ -426,27 +426,25 @@ chrome.runtime.onMessage.addListener(
         }
       });
     } else if (type === 'REQUEST_SUBMIT_TRANSACTION/V3') {
-      handleTransactionRequest({
-        payload: message.payload,
-        sender,
-        parameter: PARAMETER_SUBMIT_TRANSACTION,
-        receivingMessage: 'RECEIVE_SUBMIT_TRANSACTION/V3',
-        errorPayload: {
-          type: ResponseType.Reject,
-          result: undefined
-        }
-      });
+      const { payload } = message;
+      try {
+        sendInMemoryMessage({
+          payload,
+          parameter: PARAMETER_SUBMIT_TRANSACTION,
+          receivingMessage: 'RECEIVE_SUBMIT_TRANSACTION/V3',
+          sender
+        });
+      } catch (e) {}
```

Comment:
Tbh if it fails should we try again?
Btw what would be the reasons for this promise to fail?

Reply:
Error on the user's browser side that we do not know about (permissions, etc). idk what we can do in this case